examples/annotate-variants-job.properties

```diff
@@ -0,0 +1,45 @@
+# JOB
+spring.batch.job.names=annotate-variants-job
+
+# SUBMISSION FIELDS
+input.vcf=/path/to/input.vcf
+input.vcf.id=1234
+input.vcf.aggregation=NONE
+
+input.study.name=Study name
+input.study.id=2345
+input.study.type=COLLECTION
+
+input.pedigree=
+input.fasta=/path/to/homo_sapiens/sequence.fa
```

Comment:
The following input properties shouldn't be necessary:

* input.vcf
* input.vcf.id
* input.vcf.aggregation
* input.study.name
* input.study.type
* input.pedigree